server/app/Http/Requests/UsersRequest/UserCreateRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Requests\UsersRequest;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserCreateRequest extends UserLoginRequest
+{
+    private function SetNameRequest()
+    {
+        $name = array([
```

Comment:
🐛 **BUG**: La regla de validación `name` tiene una estructura incorrecta, por lo que no se valida el campo nombre provocando el error: `PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null`

```sh
# UserCreateRequest.php > rules()
{
  "email": [
    "bail",
    "required",
    "email",
    "max: 80",
    "unique:cliente_usuario,email"
  ],
  "password": ["bail", "required", "string", "min: 8", "confirmed"],
  "0": [{ "name": ["bail", "required", "string", "max: 50"] }]  # <-- Acá esta el problema
}
```